modules/assim.batch/R/minimize.GP.R

```diff
@@ -329,7 +327,7 @@ mcmc.GP <- function(gp, x0, nmcmc, rng, format = "lin", mix = "joint", splinefcn
       for (i in seq_len(dim)) {
         ## propose new
         repeat {
-          xnew[i] <- rnorm(1, xcurr[[i]], p(jmp)[i])
+          xnew[i] <- stats::rnorm(1, xcurr[[i]], PEcAn.emulator::p(jmp)[i])
```

Comment:
@mdietze This line complains "no visible binding for global variable 'jmp'", apparently because all earlier assignments to `jmp` (lines 257, 268) are commented out. This is the last remaining error blocking CI -- Can you suggest how to resolve?

Reply:
OK, so it looks like @istfer commented out all my jmp uses ~6 years ago and instead was using jcov as the jump covariance. She preserved what I'd done previously, so we could go back if we want/need to, so I'd recommend you commend out the line and add this duplicate, which should work
```
 xnew[i] <- stats::rnorm(1, xcurr[[i]], diag(jcov)[i])
```

Reply:
```suggestion
          # xnew[i] <- stats::rnorm(1, xcurr[[i]], PEcAn.emulator::p(jmp)[i])
           xnew[i] <- stats::rnorm(1, xcurr[[i]], diag(jcov)[i])
```